postgis/postgis.sql.in

```diff
@@ -2860,6 +2860,18 @@ CREATE OR REPLACE FUNCTION ST_Transform(geom geometry, from_proj text, to_srid i
 	LANGUAGE 'sql' IMMUTABLE STRICT PARALLEL SAFE
 	_COST_HIGH;
 
+CREATE OR REPLACE FUNCTION postgis_spatial_ref_sys(auth_name_in text, auth_srid_in text, OUT auth_name text, OUT auth_srid TEXT, OUT srtext TEXT, OUT proj4text TEXT)
+	RETURNS RECORD
+	AS 'MODULE_PATHNAME', 'postgis_srs_entry'
+	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE
+	_COST_MEDIUM;
+
+CREATE OR REPLACE FUNCTION postgis_spatial_ref_sys(OUT auth_name text, OUT auth_srid TEXT, OUT srtext TEXT, OUT proj4text TEXT)
+	RETURNS SETOF RECORD
```

Comment:
@strk this doesn't mirror spatial_ref_sys table unless you are suggesting @pramsey add an srid there.
That doesn't feel right though because as mentioned some auth_srid are not integers and our convention is to use the auth_srid as our srid.  But putting that restriction here would make it not possible to return those.

I would however like to see the more standards compliant format of below which is much less confusing than out parameters.

```
CREATE OR REPLACE FUNCTION postgis_spatial_ref_sys()
	RETURNS TABLE (auth_name text, auth_srid TEXT, srtext TEXT, proj4text TEXT)

```

I think many raster functions we had to change to the above format.  I forget why.  Something about the behavior change in CTE in PG 12.

Reply:
Returning SETOF TABLE() fails for me, hence the move to out vars.

Reply:
There is no such thing as SETOF TABLE().  It's just TABLE.  or did just TABLE fail?

Reply:
If TABLE() is implicitly set returning... I never tried.

Reply:
Yes it's implicitly set returning.